pkgs/by-name/pr/primesieve/package.nix

```diff
@@ -1,25 +1,35 @@
 { lib
-, stdenv
-, fetchFromGitHub
 , cmake
+, fetchFromGitHub
+, stdenv
+, primecount
 }:
 
-stdenv.mkDerivation rec {
+stdenv.mkDerivation (finalAttrs: {
   pname = "primesieve";
   version = "12.1";
 
   src = fetchFromGitHub {
     owner = "kimwalisch";
     repo = "primesieve";
-    rev = "v${version}";
+    rev = "v${finalAttrs.version}";
     hash = "sha256-AHl2GfZ1oJ8ZyjJzvg10AqN7TA7HFZ+qa6N2v51Qa78=";
   };
 
+  outputs = [ "out" "dev" "lib" "man" ];
```

Comment:
I guess this breaks openturns: https://hydra.nixos.org/job/nixpkgs/trunk/openturns.x86_64-linux

@AndersonTorres Could you have a look?

Reply:
@wegank 

https://github.com/NixOS/nixpkgs/pull/309351